community/iptables/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+# give option to easily enable nftables by passing USE="--with-nftables" to kiss
+[ -z "$USE" ] && USE="--disable-nftables"
```

Comment:
KISS has no USE flags. The user can simply fork the package into their own repository.

NOTE: Symlinks to the KISS repositories work for unchanged files. 

Example:

```
- build (modified to enable nftables)
- checksums -> /path/to/community/iptables/checksums
- sources   -> /path/to/community/iptables/sources
- version   -> /path/to/community/iptables/version
```

This removes the burden of maintaining the modified package yourself. :)

Reply:
agreed